pkgs/os-specific/linux/kernel/bug-null-pointer-dereference.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/mm/compaction.c b/mm/compaction.c
```

Comment:
Please add the patch header as well here. For instance, this should have `From: Suzuki K Poulose <suzuki.poulose@arm.com>` in the header. You could also do `fetchpatch` on https://lore.kernel.org/lkml/1558711908-15688-1-git-send-email-suzuki.poulose@arm.com/raw

Reply:
Updated. With fetchpatch not working.